src/frontend/web_application/src/components/MessageList/style.scss

```diff
@@ -10,10 +10,7 @@
 
   &__action {
     margin: $co-component__spacing;
-    background: $co-color__fg__back--high;
+    background: $co-color__fg__back--higher; // need a modifier for Button
```

Comment:
we need to think again "what is a button" in caliopen and refactor (and this can be done for all components)